src/com/android/volley/Request.java

```diff
@@ -699,4 +704,17 @@ public void setJoined(boolean joined) {
     public void expireCache() {
         // no - op
     }
+
+    public boolean isExecuting() {
+        return mRequestQueue.isProcessing(this);
+    }
+
+    public void setFinished(boolean finished) {
```

Comment:
Seems unnecessary to include this method. Why not use the existing `finish` method of Request and set mFinished = true there. I don't think you would ever need to set it to be false.